coordinator/src/routes/prescription/post-send-message.ts

```diff
@@ -1,8 +1,8 @@
-import * as translator from "../../services/translation/translation-service"
-import {Bundle} from "../../model/fhir-resources"
-import {defaultRequestHandler} from "../../services/spine-communication"
+import * as translator from "../../services/translation"
```

Comment:
is it possible to rename this file since its no longer `$send` its now `$process-message`.